templates/includes/icons/sprites.html

```diff
@@ -31,6 +31,15 @@
         <path fill="currentColor" d="M505 442.7L405.3 343c-4.5-4.5-10.6-7-17-7H372c27.6-35.3 44-79.7 44-128C416 93.1 322.9 0 208 0S0 93.1 0 208s93.1 208 208 208c48.3 0 92.7-16.4 128-44v16.3c0 6.4 2.5 12.5 7 17l99.7 99.7c9.4 9.4 24.6 9.4 33.9 0l28.3-28.3c9.4-9.4 9.4-24.6.1-34zM208 336c-70.7 0-128-57.2-128-128 0-70.7 57.2-128 128-128 70.7 0 128 57.2 128 128 0 70.7-57.2 128-128 128z"> 
     </symbol>
     <symbol id="external-link" viewBox="0 0 18 18" fill="currentColor">
-        <path fill="currentColor" d="M16 16H2V2h7V0H2a2 2 0 0 0-2 2v14a2 2 0 0 0 2 2h14c1.1 0 2-.9 2-2V9h-2v7ZM11 0v2h3.59l-9.83 9.83 1.41 1.41L16 3.41V7h2V0h-7Z"/>
+        <path fill="currentColor" d="M16 16H2V2h7V0H2a2 2 0 0 0-2 2v14a2 2 0 0 0 2 2h14c1.1 0 2-.9 2-2V9h-2v7ZM11 0v2h3.59l-9.83 9.83 1.41 1.41L16 3.41V7h2V0h-7Z" />
+    </symbol>
+    <symbol id="calendar" viewBox="0 0 19 21" fill="none">
```

Comment:
Are these the icons that have come from marketing?

Reply:
From the Figma designs, but I've spoken to Nick about this as we've been using google material icons in the designs and font awesome in the front-end, so for simplicity I think we're going to update the designs with font awesome icons moving forward. I've removed these svgs now and replaced with font awesome.

Reply:
Okay. Hold fire on that migration to Font Awesome though. I have been finding Font Awesome too bulky and I wanted to have a conversation with Simon and the marketing team because although I initially used Font Awesome because it was easy and I thought that's what we were using, I think I would actually prefer to use the Material icons set like I have done in the past.

Reply:
I _might_ suggest that we keep using the brand icons from Font Awesome but all the other icons from Material...

Reply:
TL;DR - use FA for now, don't change Figma and I can update them in `tna-frontend` later 👍🏼